src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/clusterinteraction/ClusterInteractionFragment.kt

```diff
@@ -109,7 +94,7 @@ class ClusterInteractionFragment : Fragment() {
   inner class EndpointListener : EndpointAdapter.OnItemClickListener {
     override fun onItemClick(position: Int) {
       Toast.makeText(requireContext(), "Item $position clicked", Toast.LENGTH_SHORT).show()
-      showFragment(ClusterDetailFragment.newInstance())
+      showFragment(ClusterDetailFragment.newInstance(clusterMap as HashMap<String, ClusterInfo>, devicePtr))
```

Comment:
Passing `devicePtr` makes sense because you pick the node ID in this screen, but why pass clusterMap when you could just do `ClusterInfoMapping().clusterMap` directly in the other fragment?